test/Microsoft.AspNetCore.Routing.Tests/Matchers/TreeRouterMatcherConformanceTest.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Routing.Internal;
+using Microsoft.AspNetCore.Routing.Template;
+using Microsoft.AspNetCore.Routing.Tree;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.ObjectPool;
+using Microsoft.Extensions.Options;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Routing.Matchers
+{
+    public class TreeRouterMatcherConformanceTest : MatcherConformanceTest
+    {
+        [Fact(Skip= "OH NOES unsupported")]
+        public override Task Match_SingleLiteralSegment_Success()
```

Comment:
This is an example of how we can handle divergences in behavior. 

You can override and skip if it's a temporary issue, or override and noop if it's a known difference in expectations.

This also allows us to add more targeted tests for specific implementations if we so desire.